sway-lib-std/src/token.sw

```diff
@@ -6,6 +6,21 @@ use ::contract_id::ContractId;
 use ::revert::revert;
 use ::tx::*;
 use ::context::call_frames::contract_id;
+use ::identity::Identity;
+
+/// Mint `amount` coins of the current contract's `asset_id` and send them to `destination` by calling either force_transfer() or transfer_to_output(), depending on the type of `Identity`.
+pub fn mint_to(amount: u64, recipient: Identity) {
+    mint(amount);
+    match recipient {
+        Identity::Address(addr) => {
+            transfer_to_output(amount, contract_id(), addr);
+        },
+        Identity::ContractId(id) => {
+            force_transfer(amount, contract_id(), id);
```

Comment:
`force_transfer()` really needs to be renamed to `transfer_to_contract()` to follow the similar name of transferring to an output

Reply:
`transfer_to_output()` can be stopped?

Reply:
no, but forwarding coins to a contract via a call could be rejected. This is a way to force coins directly to a contract without the receiving contract being able to do about it.
The danger here is that the receiving contract may not be equipped to ever withdraw/move those coins so they can potentially be lost.

Reply:
`transfer_to_contract` alone isn't the safest name because it doesn't capture the semantics that it's _only_ transferring to a contract, with no possibility of logic execution by the receiver. This doesn't really exist in Ethereum land, so we have to be careful. Regardless, a name change can be bikeshed in a subsequent PR.

Reply:
>a name change can be bikeshed in a subsequent PR.
Agreed. I almost changed it here, but reverted the commit.